client/src/theme/theme.ts

```diff
@@ -1,11 +1,56 @@
-/**
- * Add styles to the theme.
- */
 export const theme = {
   colors: {
-    primary: "#FF9900",
-    secondary: "#FF3D00",
-    highlighted: "#00FF0A",
-    pageBackground: "#eeffff",
+    common: {
+      textColor: "#000",
+      textInactive: "#A59E9E",
+      backgroundcolor: "#FFF",
+    },
+    navigationBar: {
+      primary: "#73A942",
+      secondary: "#D7FFA6", 
+    },
+    card: {
+      primary: "#F8F3EF",
+      secondary: "#F9F8F8",
+      highlighted: "#EEE6E6",
+      boxShadow: "0px 4px 4px 0px rgba(0, 0, 0, 0.25)",
+    },
+    button: {
+      primary: "#538D22",
+      secondary: "#F9F8F8",
+      textInactive: "#8B8B8B",
+      hover: "#AAD576",
+    },
+  },
+
+};
+
+export const text = {
```

Comment:
I am deeply convinced that everything we want to be in the theme should only be in this variable

Reply:
I move them to the theme variable, but I think that I should move them to typography?